Bioinf_Utility/modules/dna_rna_tools.py

```diff
@@ -0,0 +1,84 @@
+# creating a dictionary for the complementary DNA sequence
+complement_dict = {
+    "A": "T",
+    "a": "t",
+    "G": "C",
+    "g": "c",
+    "T": "A",
+    "t": "a",
+    "C": "G",
+    "c": "g",
+}
```

Comment:
Можно улучшить читаемость и сгруппировать в пары по верхнему-нижнему регистру: 

```suggestion
complement_dict = {
"A": "T", "a": "t",
    "G": "C", "g": "c",
    "T": "A", "t": "a",
    "C": "G", "c": "g",
}
```

Reply:
это так меняет black)) мне честно самой больше нравится, когда сгруппировано)